docs/specs/clients/notify/rpc-methods.md

```diff
@@ -1,33 +1,129 @@
 # RPC Methods
 
-This doc should be used as a _source-of-truth_ and reflect the latest decisions and changes applied to the WalletConnect collection of client-to-client JSON-RPC methods for all platforms SDKs.
+Methods can be called after the client [registers an Identity Key](../../servers/keys/identity-keys.md#key-registration).
+
+All methods contain an "auth" field in the `params` (for request) or `result` (for response) which is a signed JWT. JWTs are specified in the [authentication](./authentication.md) document.
+
+All methods follow the [JSON-RPC 2.0 spec](https://www.jsonrpc.org/specification).
+
+## Getting & watching subscriptions
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant C as Client
+  participant N as Notify Server
+
+  activate C
+  C->>+N: Get did.json
+  N-->>-C: did.json
+  Note over C: Derive req topic from did.json
```

Comment:
should we be more explicit about what it means?
```suggestion
  Note over C: Derive req topic by hashing keyAgreement key from Notify Server's did.json
```

Reply:
Don't want to make the note too wide